server/app.py

```diff
@@ -284,6 +275,68 @@ def get_status(job_id):
     return Response[JobStatusResponse](data=JobStatusResponse(status=status))
 
 
+def aggregate_statuses(input_job_id: str, output_job_id: str):
+    input_job_status = process_file_streaming.AsyncResult(input_job_id).status
+    output_job_status = process_streaming_output.AsyncResult(output_job_id).status
+
+    statuses = [input_job_status, output_job_status]
+
+    if "PENDING" in statuses:
+        return "PENDING"
+    if "FAILURE" in statuses:
+        return "FAILURE"
+    if "REVOKED" in statuses:
+        return "REVOKED"
+    if "STARTED" in statuses or "RETRY" in statuses:
+        return "STARTED"
+
+    return "SUCCESS"
+
+
+@app.get("/streaming-jobs/{job_id}", response_model=Response[JobStatusResponse])
```

Comment:
can we use jobs instead of streaming-jobs to be consistent ?

Reply:
we are not going to support s3 anyways

Reply:
also should update the endpoint touched by GET modelrun in LSE when streaming flag is on